core/files/fileuploader/uploader.go

```diff
@@ -362,11 +360,6 @@ func (u *uploader) setLastModifiedDate(path string) {
 	}
 }
 
-func (u *uploader) AutoType(enable bool) Uploader {
-	u.typeDetect = enable
```

Comment:
why 😿

Reply:
Nobody did anything about this flag, it is misleading :(